mod/chain-spec/pkg/chain/chain_spec.go

```diff
@@ -183,6 +183,8 @@ type Spec[
 	// GetCometBFTConfigForSlot retrieves the CometBFT config for a specific
 	// slot.
 	GetCometBFTConfigForSlot(slot SlotT) CometBFTConfigT
+
+	GetValidatorsSetCapSize() uint32
```

Comment:
```suggestion
        // GetValidatorSetCap retrieves the maximum number of validators allowed in the active set.
	GetValidatorSetCap() uint32
```
nit + comment

Reply:
It seems comment was changed, but function name still needs to be changed to `GetValidatorSetCap`

Reply:
indeed, fixed